### PR TITLE
[Testing] Re-enable test validation during PR validation on github #68

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: .github/workflows/angular.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - run: npm run build
-    #- run: npm run test
+    - run: npm run test
```
Comment (MSNev) [inline .github/workflows/angular.yml:29]: Re-enabling tests during PR validation

File: package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --configuration production",
-    "test": "ng test",
+    "test": "ng test --watch=false",
```
Comment (MSNev) [inline package.json:17]: Re-enabling tests during PR validation and still providing a `watch` option

File: .github/workflows/angular.yml (hunk 1 above)
Comment (timdeschryver) [reply .github/workflows/angular.yml:29]: I also noticed that the workflow uses `actions/checkout@v2` while `actions/checkout@v3` is stable.
Should that also be updated?

Comment (MSNev) [reply .github/workflows/angular.yml:29]: Good point

**Review by Karlie-777 — APPROVED** [review]

**Pull request merged**